frontend/src/styles/HetComponents/HetTags.tsx

```diff
@@ -1,19 +1,28 @@
+interface Tag {
+  name: string
+  link?: string
+}
+
 interface HetTagsProps {
-  tags: string[]
+  tags: string[] | Tag[]
```

Comment:
Are we able to transform plain strings into `Tags` before they reach this point?